web_src/css/repository.css

```diff
@@ -1846,6 +1846,24 @@
   padding-bottom: 5px;
 }
 
+.diff-file-box {
+  border: 1px solid transparent;
+  border-radius: 0.285rem; /* Just like ui.top.attached.header */
+}
+
+.diff-file-box .ui.attached.header,
+.diff-file-box .ui.attached.table {
+  margin: 0; /* remove fomantic negative margins */
+  width: initial; /* remove fomantic over 100% width */
+  max-width: initial; /* remove fomantic over 100% width */
+}
```

Comment:
Should try later to make these styles global. Fomantic's use of -1 left/right margin and 100% + 2px width on these `attached` fomantic elements makes certain effects like border impossible because it makes elements break outside their container.